waveformCompare.py

```diff
@@ -2129,19 +2129,19 @@ def generate_tags(event):
     time_tag = orig['time'].isoformat()[:19].replace(':','')+'Z'
 
     #. Organize in Year and Monthly folders, i.e. '2017/09/GCMT_2017-09-23T125302_6.05_OAXACA_MEXICO'
-    output_path_alt = '{}/{}/{}'.format(output_path, str(orig['time'].year),
-                                        str(orig['time'].month))
+    output_path_subtree = '{}/{}/{}'.format(output_path, str(orig['time'].year),
+                                        (str(orig['time'].month)).zfill(2))

```

Comment:
Ah.. my bad.. should've thought about that when proposing that change more. Actually, it's easier and more readable to specify that format like..

`'{}/{:4d}/{:02d}'.format(output_path, orig['time'].year, orig['time'].month)`

And please watch indentation level with continuation lines, that's not PEP8 compatible in the above case: https://www.python.org/dev/peps/pep-0008/#indentation

Like mentioned earlier, please use  e.g. `flake8` for checking code layout.